DID.as

```diff
@@ -104,6 +109,7 @@ namespace DID {
 
     void OnSettingsChanged() {
         init();
+        SetLaneProvidersFromSettings();
```

Comment:
put in init() pls?

Reply:
I don't think it's needed in init. Will add it b/c the overhead is negligible